### ISSUE TITLE
Confirm customer

### ISSUE THREAD

**Issue by cmuench** [netz98/n98-magerun2#522]
If a new customer is created an email to confirm the new account should be send.
Sometimes this mail does not arrive.
A command `customer:confirm` could be handy.

**Comment by Serializator** [comment]
What if you're working on the confirm e-mail or debugging it? 
What about a flag (`--resend` or `--retry`) that doesn't confirm the account but tries to send the e-mail again?
If the above were to be added, any ideas what the most logical would be to do if the account is already confirmed, simply an error message or maybe the question to un-confirm the account before retrying?

**Comment by cmuench** [comment]
@Serializator Good idea. The command is currently not existing and hast to be ported from M1 to M2.
